docs/source/cli_help/format.rst

```diff
@@ -23,8 +23,8 @@ In the above example ``develop`` and ``master`` are roots of the tree of branch
 Branches ``adjust-reads-prec``, ``edit-margin-not-allowed`` and ``grep-errors-script`` are direct downstream branches for ``develop``.
 ``block-cancel-order`` is a downstream branch of ``adjust-reads-prec``, ``change-table`` is a downstream branch of ``block-cancel-order`` and so on.
 
-Every branch name can be followed (after a single space as a delimiter) by a custom annotation --- a PR number in the above example.
-The annotations don't influence the way ``git machete`` operates other than that they are displayed in the output of the ``status`` command.
+Every branch name can be followed (after a single space as a delimiter) by a custom annotation, e.g. *PR #234*, *PR #235 rebase=no* or *push=no*.
+Annotations might contain underlined branch qualifiers (``rebase=no``, ``push=no``) that control rebase and push behaviour of ``traverse`` (see help for :ref:`traverse`)
```

Comment:
```suggestion
Annotations might contain underlined branch qualifiers (``rebase=no``, ``push=no``) that control rebase and push behaviour of ``traverse`` (see help for :ref:`traverse`).
```

I'd also add such a reference "(see help for :ref:`traverse`)" in other places where the qualifiers are mentioned ✍🏻